src/routes/about/index.svelte

```diff
@@ -42,36 +50,78 @@
 </script>
 
 <style lang="scss">
+  figure {
+    margin: 0;
+
+    @media (max-width: 1052px) {
```

Comment:
Note: these hard-coded breakpoints in pixels correspond to the Carbon Grid breakpoints.

It's annoying that you can't currently use CSS variables with media queries, although supposedly a future version of media queries will support using them. In the meantime I'm wondering if they are exposed in the Carbon Design system Sass?